commands/update/cluster/command.go

```diff
@@ -18,20 +18,23 @@ import (
 	"github.com/giantswarm/gsctl/flags"
 )
 
+const labelFilterKeySubstring = "giantswarm.io"
```

Comment:
Not a big deal but is there a place where this can be defined once instead of in every command?